packages/console/src/assets/docs/guides/m2m-general/README.mdx

```diff
@@ -1 +1,110 @@
-## Replace this with actual guide
\ No newline at end of file
+import InlineNotification from '@/ds-components/InlineNotification';
+import Steps from '@/mdx-components-v2/Steps';
+import Step from '@/mdx-components-v2/Step';
+import ApplicationCredentials from '@/mdx-components-v2/ApplicationCredentials';
+import EnableAdminAccessSrc from './assets/enable-admin-access.png';
+import AppIdentifierSrc from './assets/api-identifier.png';
+
+<Steps>
+<Step title="Intro">
+Machine to Machine (M2M) is a common practice to authenticate if you have an app that needs to directly talks to resources. E.g., an API service that updates users' custom data in Logto, a statistic service that pulls daily orders, etc.
+
+Usually, an M2M app doesn't need user interactions, i.e., it has no UI.
+
+</Step>
+<Step title="Locate the app ID and app secret" subtitle="1 step">
+Get your App ID and App Secret.
+
+<ApplicationCredentials id={props.app.id} secret={props.app.secret} />
+
+</Step>
+<Step title="Locate the API Resource" subtitle="2 steps">
+
+### Accessing Logto Management API
+
+If you want to use this app for accessing [Management API](https://docs.logto.io/docs/references/core/#management-api), remember to turn on "Enable admin access" in the Advanced settings tab.
+
+<img alt="API identifier" src={EnableAdminAccessSrc} width="600px" />
+
+### Find the API identifier
+
+In the API Resource tab, find the API identifier that the app needs to access. If you haven't added the API Resource in Logto or don't know what API Resource is, see [API Resource](https://docs.logto.io/docs/references/resources).
+
+<img alt="API identifier" src={AppIdentifierSrc} width="600px" style={{ paddingBottom: '12px' }} />
```

Comment:
this is ambiguous. do i need to find the api identifier if i wang to access logto management api? should be more clear

Reply:
updated, make it a separate optional step, ahead of the locate API identifier.